backend/src/api/monthReports/monthReport.controller.js

```diff
@@ -0,0 +1,57 @@
+const catchAsync = require('../../utils/catchAsync');
+const MonthReportModel = require('../cron/monthReport.model');
+const AppError = require('../errors/appError');
+
+const getMonthReport = catchAsync(async (req, res, next) => {
+  let { startMonth, startYear } = req.query;
+
+  if (Object.keys(req.query).length === 0) {
```

Comment:
First of all, there is a package, called joi. And we decided to use it instead of `if`'s validation